--- src/Makefile.am.link	2011-05-10 03:37:04.000000000 +0200
+++ src/Makefile.am	2011-05-10 03:37:41.000000000 +0200
@@ -39,6 +39,7 @@
 else
 libunicap_la_LIBADD = \
 	@DL_LIBS@ \
+	@PTHREAD_LIBS@ \
 	@M_LIBS@
 endif
 
--- src/Makefile.in.link	2011-05-10 03:37:11.000000000 +0200
+++ src/Makefile.in	2011-05-10 03:38:08.000000000 +0200
@@ -272,6 +272,7 @@
 @BUILD_VID21394_TRUE@LIBVID21394 = cpi/vid21394/libvid21394.la
 @ENABLE_STATIC_CPI_FALSE@libunicap_la_LIBADD = \
 @ENABLE_STATIC_CPI_FALSE@	@DL_LIBS@ \
+@ENABLE_STATIC_CPI_FALSE@	@PTHREAD_LIBS@ \
 @ENABLE_STATIC_CPI_FALSE@	@M_LIBS@
 
 @ENABLE_STATIC_CPI_TRUE@libunicap_la_LIBADD = \
--- cpi/euvccam/Makefile.am.link	2011-05-10 03:44:01.000000000 +0200
+++ cpi/euvccam/Makefile.am	2011-05-10 03:44:43.000000000 +0200
@@ -5,7 +5,7 @@
 libeuvccam_cpi_la_LIBADD=-lpthread 
 else
 libcpi_LTLIBRARIES=libeuvccam_cpi.la
-libeuvccam_cpi_la_LIBADD=-lpthread -L../../src/.libs/ -lunicap
+libeuvccam_cpi_la_LIBADD=-lpthread -L../../src/.libs/ -lunicap @RT_LIBS@
 endif
 libcpidir = $(libdir)/unicap$(pkg_version)/cpi
 
--- cpi/euvccam/Makefile.in.link	2011-05-10 03:44:14.000000000 +0200
+++ cpi/euvccam/Makefile.in	2011-05-10 03:44:57.000000000 +0200
@@ -260,7 +260,7 @@
 unicap_minor_version = @unicap_minor_version@
 INCLUDES = -I../include -I../../include -I../../ -I../../common
 @ENABLE_STATIC_CPI_TRUE@noinst_LTLIBRARIES = libeuvccam_cpi.la
-@ENABLE_STATIC_CPI_FALSE@libeuvccam_cpi_la_LIBADD = -lpthread -L../../src/.libs/ -lunicap
+@ENABLE_STATIC_CPI_FALSE@libeuvccam_cpi_la_LIBADD = -lpthread -L../../src/.libs/ -lunicap @RT_LIBS@
 @ENABLE_STATIC_CPI_TRUE@libeuvccam_cpi_la_LIBADD = -lpthread 
 @ENABLE_STATIC_CPI_FALSE@libcpi_LTLIBRARIES = libeuvccam_cpi.la
 libcpidir = $(libdir)/unicap$(pkg_version)/cpi
